docs/utilities/data_classes.md

```diff
@@ -1121,3 +1120,28 @@ This example is based on the AWS Blog post [Introducing Amazon S3 Object Lambda
         for record in event.records:
             do_something_with(record.body)
     ```
+
+## Advanced
+
+### Debugging
+
+Alternatively, you can print out the fields to obtain more information. All classes come with a `__str__` method that generates a dictionary string which can be quite useful for debugging.
+
+However, certain events may contain sensitive fields such as `secret_access_key` and `session_token`, which are labeled as sensitive to prevent any accidental disclosure of confidential information.
```

Comment:
!!! info "`[SENSITIVE]`"

Reply:
My suggestion:
> !!! info "Some events contain sensitive fields, such as as credentials (access keys, passwords, tokens). These fields appear as `[SENSITIVE]` to prevent accidental dislosure of confidential information."

On this note, do we want an environment variable to override this behaviour when a developer wants complete information for deeper debugging?

Reply:
The developer will always have the chance to print the specific event field, for example: `logger.info(event.session_token)`. I think that adding an environment variable brings complexity and the possibility of errors, better to keep it simple and functional as it is.